### PR TITLE
Handle long program names

### PR DESCRIPTION
**Pull request by cce**
Papertrail supports long APP-NAME syslog messages, so this vendors in the syslog_protocol package and modifies it to support program names longer than 32 characters.

Includes #32 with a fix from @mariozski where logs didn't appear.

### CODE REVIEW

**Comment by trevrosen** [comment]
@cce I approved and GitHub made my now-irrelevant comments front-and-center, so... ignore them

File: fluent-plugin-papertrail.gemspec [hunk 1]
```diff
@@ -22,7 +22,6 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "fluentd", '>= 0.10', '< 2'
   spec.add_dependency "fluent-mixin-config-placeholders", "~> 0.4.0"
-  spec.add_dependency "syslog_protocol"
```
[the remark this answers is not in the document]
Comment (cce) [reply fluent-plugin-papertrail.gemspec]: No longer using a gem, added to /lib

**Comment by cmrust** [comment]
👋

**Comment by leecalcote** [comment]
He lives! ;)

**Comment by cce** [comment]
Fixed by @mariozski in #32.

**Pull request merged**